### PR TITLE
Bug 1753559: jsonnet: increase the 'for' clause for timestamp alerts

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

**Comment by brancz** [comment]
What speaks against doing this change upstream?

**Comment by simonpasquier** [comment]
> What speaks against doing this change upstream?

Choosing a default `for` value is tricky as there's no one-size-fits-all. I felt that in a general context, `1h` wouldn't be more "correct" than `10min`.

**Comment by brancz** [comment]
I don't feel too strongly, but I feel 10m is generally too short.

File: assets/prometheus-k8s/rules.yaml [hunk 1]
```diff
@@ -1093,7 +1093,7 @@ spec:
         summary: Prometheus is dropping samples with duplicate timestamps.
       expr: |
         rate(prometheus_target_scrapes_sample_duplicate_timestamp_total{job="prometheus-k8s",namespace="openshift-monitoring"}[5m]) > 0
-      for: 10m
+      for: 1h
```
[the remark this answers is not in the document]
Comment (simonpasquier) [reply assets/prometheus-k8s/rules.yaml]: I didn't think a lot about the new value, just that 1 hour should be enough for the upgrade to complete.

**Comment by simonpasquier** [comment]
> Also from what I understand from the bugzilla do we need to change this in 4.1 as well? As this happens during the upgrade, which rule will then be taken?

That's a good point. I suspect that indeed this has to be backported to 4.1.

File: assets/prometheus-k8s/rules.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (LiliC) [reply assets/prometheus-k8s/rules.yaml]: If it works fine by me :) 👍

**Comment by brancz** [comment]
Yes I think we should just always do 30m here and upstream and see how it works out.

**Comment by brancz** [comment]
I'm happy to have us move forward here though and follow up upstream once we have validated that it's actually the better value.

/lgtm

**Comment by simonpasquier** [comment]
Re backporting to 4.1, it isn't needed because the alerts didn't exist in 4.1 :relieved:

**Pull request merged**